plateau/io/dask/dataframe.py

```diff
@@ -584,7 +584,8 @@ def collect_dataset_metadata(
             # ensure that even with sampling at least one metapartition is returned
             cutoff_index = max(1, int(len(mps) * frac))
             mps = mps[:cutoff_index]
-            ddf = dd.from_delayed(
+            ddf = dd.from_map(
+                lambda x: x.compute(),
```

Comment:
@fjetter is this a reasonable way of converting `from_delayed` to `from_map`? Should I instead be implementing a subclass of `DataFrameIOFunction` as you did in https://github.com/data-engineering-collective/plateau/pull/80?

Reply:
Discussed offline, short answer: no, this is not the way to convert `from_delayed` to `from_map`.